docs/commits_and_prs.md

```diff
@@ -0,0 +1,105 @@
+Writing Commits and Pull Requests
+=================================
+
+Small, atomic commits and pull requests (PRs) with helpful commit messages,
+PR titles and PR bodies are great for collaboration and key to a maintainable
+project. Tools like [`git rebase`](https://docs.github.com/en/get-started/using-git/using-git-rebase-on-the-command-line)
```

Comment:
The [Rewriting History](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) section of the Git book and <https://git-rebase.io/> might be better links here

Reply:
Changed the link to https://git-rebase.io/